apps/src/lab2/views/dialogs/StartOverDialog.tsx

```diff
@@ -11,13 +13,21 @@ const StartOverDialog: React.FunctionComponent<BaseDialogProps> = ({
   handleConfirm,
   handleCancel,
 }) => {
+  const currentAppName = useAppSelector(
```

Comment:
@sanchitmalhotra126 I'm pretty confident this won't impact Music Lab since we are only using the new translated string if we are in a text-based lab. LMK if this is too risky during the launch period.